main.cc

```diff
@@ -965,14 +965,14 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
 
             // Note: changed from using a move here, because we want the config object intact.
             replica::database_config dbcfg;
-            dbcfg.compaction_scheduling_group = make_sched_group("compaction", 1000);
-            dbcfg.memory_compaction_scheduling_group = make_sched_group("mem_compaction", 1000);
+            dbcfg.compaction_scheduling_group = make_sched_group("compaction", "comp", 1000);
+            dbcfg.memory_compaction_scheduling_group = make_sched_group("mem_compaction", "mcmp", 1000);
             dbcfg.streaming_scheduling_group = maintenance_scheduling_group;
-            dbcfg.statement_scheduling_group = make_sched_group("statement", 1000);
-            dbcfg.memtable_scheduling_group = make_sched_group("memtable", 1000);
-            dbcfg.memtable_to_cache_scheduling_group = make_sched_group("memtable_to_cache", 200);
-            dbcfg.gossip_scheduling_group = make_sched_group("gossip", 1000);
-            dbcfg.commitlog_scheduling_group = make_sched_group("commitlog", 1000);
+            dbcfg.statement_scheduling_group = make_sched_group("statement", "stmt", 1000);
+            dbcfg.memtable_scheduling_group = make_sched_group("memtable", "mt", 1000);
+            dbcfg.memtable_to_cache_scheduling_group = make_sched_group("memtable_to_cache", "mm2c", 200);
```

Comment:
`mt2c`, to be consistent with the `mt` above (and the abbreviations used in the code already)

Reply:
@denesb thank you Botond for the explanation! and sorry, i was somehow thinking about the memory subsystem of linux at that moment, but missed the table part!